src/Projections/Benchmark/Benchmark.csproj

```diff
@@ -5,6 +5,10 @@
     <Platforms>x64;x86</Platforms>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
does this not support .NET 5?

Reply:
Right we only use the patcher on net5 assemblies